pkgs/development/python-modules/dnslib/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "dnslib";
+  version = "0.9.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0r7q7rgm9dymg7ka4cypzar83c5sagdrfda3fcy5xj0dd7j9kn4q";
+  };
+
+  # Tests are broken on Python 3.7.
+  doCheck = false;
```

Comment:
```suggestion
  doCheck = isPy37;
```
What's the error?

Reply:
The package has a non-standard way to run tests and it just turns out that the default checkPhase broke on py3. I fixed it properly now!